index.js

```diff
@@ -13,7 +13,7 @@ var events = require('events');
 var Q = require('q');
 var nunjucks = require('nunjucks');
 var util = require('./src/util/util');
-
+var highAvailability = process.env.HIG_AVAILABILITY;
```

Comment:
Can we rename this flag?... I would prefer if it were more descriptive of what is going on. Maybe ```MONGO_HIGH_AVAILABILITY``` or something like that would be good for me.